setup.py

```diff
@@ -50,10 +57,12 @@ def run(self):
     url='https://github.com/ua-parser/uap-python',
     include_package_data=True,
     package_data={'ua_parser': ['regexes.yaml', 'regexes.json']},
+    setup_requires=['pyyaml'],
     install_requires=['pyyaml'],
```

Comment:
Changing this line from `install_requires=['pyyaml']` to `tests_require=['pyyaml']` would let users choose not to install pyyaml if they only want to use the parser, since user_agent_parser.py doesn't use pyyaml, but it would still be installed if `python setup.py test` was run.

The only other pyyaml usage was in the setup.py itself, and I had to add setup_requires to ensure it was available to the setup.py script.